src/Illuminate/Queue/QueueManager.php

```diff
@@ -33,15 +34,24 @@ class QueueManager implements FactoryContract, MonitorContract
      */
     protected $connectors = [];
 
+    /**
+     * The shared data that should be added to the queue payload.
+     *
+     * @var \Illuminate\Support\Collection
+     */
+    protected $shared;
+
     /**
      * Create a new queue manager instance.
      *
      * @param  \Illuminate\Foundation\Application  $app
+     * @param  \Illuminate\Support\Collection  $shared
```

Comment:
Shouldn't this be `\Illuminate\Support\Collection|null`?

Reply:
Yes it should will update